### PR TITLE
Feature/anra6 nrlf 494 create excludes special characters

### PR DESCRIPTION
**Pull request by andrewrae5-nhs**
https://nhsd-jira.digital.nhs.uk/browse/NRLF-494 - JIRA ticket

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -569,3 +569,37 @@ When you create a release branch in the form of `release/yyyy-mm-dd` or `hotfix/
 The CI pipeline will check to make sure you have done this to prevent any mistakes - if you have made a release or hotfix branch it will check that the value in the RELEASE file matches or not
 
 This is because it will use that value to tag the commit once its been merged into main as a reference point, and this is how it tracks which release it is as github actions struggles with post merge branch identification
+
```
Comment (jaklinger) [inline README.md:572]: worth amending and duplicating parts of this with the section `4. Run the Feature Tests`

File: api/producer/record-locator/producer.yaml [hunk 2]
```diff
@@ -1000,7 +1000,7 @@ components:
             - DocumentReference
         id:
           type: string
-          pattern: "[A-Za-z0-9\\-\\.]{1,64}"
+          pattern: "[A-Za-z0-9\\.]+-[A-Za-z0-9]+[A-Za-z0-9\\_\\-]+"
```
Comment (jaklinger) [inline api/producer/record-locator/producer.yaml:1003]: I think the `{1,64}` bit was important to keep the id length to 64 chars

File: api/producer/swagger.yaml [hunk 3]
```diff
@@ -1097,7 +1097,7 @@ components:
             - DocumentReference
         id:
           type: string
-          pattern: "[A-Za-z0-9\\-\\.]{1,64}"
+          pattern: "[A-Za-z0-9\\.]+-[A-Za-z0-9]+[A-Za-z0-9\\_\\-]+"
```
Comment (jaklinger) [inline api/producer/swagger.yaml:1100]: as above - need to keep the `{1,64}`

File: README.md (hunk 1 above)
Comment (andrewrae5-nhs) [reply README.md]: moved into section 4

**Pull request merged**